libs/data-types/src/lib/decorators/Union.ts

```diff
@@ -0,0 +1,30 @@
+import { UnionMetadata } from '../enums/types/UnionMetadata';
+import { getCoscradDataSchema } from '../utilities';
+import appendMetadata from '../utilities/appendMetadata';
+import getDiscriminantForUnionMember from '../utilities/getDiscriminantForUnionMember';
+import mixinDefaultTypeDecoratorOptions from './common/mixinDefaultTypeDecoratorOptions';
+import { TypeDecoratorOptions } from './types/TypeDecoratorOptions';
+
+/**
+ * TODO We should have a test that validates the `discriminantPath` or else
```

Comment:
Tag this with the following [story](https://www.pivotaltracker.com/story/show/183320394).

Reply:
tagged